spec/controllers/equipment_controller_spec.rb

```diff
@@ -10,13 +10,13 @@
 
     describe 'POST update' do
       let!(:equipment1) { create :item,
-          loader: "equipment.weapons.laser_gatling",
           spaceship: user.active_spaceship,
+          loader: "equipment.weapons.laser_gatling",
```

Comment:
Why that change?

Reply:
It's a useful base-class for applications services that provides a thin layer of attribute checking -- i incorporated the source into `ApplicationService` so the gem wouldn't have to be included.

Reply:
put it back...don't know why it was moved.